### ISSUE TITLE
Ignoring of the name parameter of field attribute

### ISSUE THREAD

**Issue by qnixdev** [overblog/GraphQLBundle#1163]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no
| Version| 0.15.2

Hello all!
I'm trying to change the name of a field, but I'm getting a name error...
In the parameter I explicitly specified the property name, but it is ignored.
I have a transformer that converts a value to a timestamp before creating an object (implementation omitted for brevity).

```php
use Overblog\GraphQLBundle\Annotation as GQL;

#[GQL\Input(name: 'MyInput')]
class MyInputType
{
    #[GQL\Field(name: 'endTime', type: 'String!')]
    private int $endTimestamp;
}
```

```json
{
    "message": "Variable \"$input\" got invalid value {\"endTime\":\"18:00\"}; Field value.endTimestamp of required type String! was not provided.",
    "extensions": {
        "category": "graphql"
    },
    "locations": [
        {
            "line": 1,
            "column": 24
        }
    ]
}
```

**Comment by Vincz** [comment]
Hey @qnixdev! Looks like a cache problem to me. Did you check the generated `Input`?

**Comment by qnixdev** [comment]
> Hey @qnixdev! Looks like a cache problem to me. Did you check the generated `Input`?

Yes of course I checking the generated `Input`. Also I'am cleared all the cache, but this didn't help((
If open the `/graphiql` documentation, the property name will also be generated from the field name..

**Comment by Vincz** [comment]
Hi @qnixdev Any news about this? Have you been able to fix it?
